packages/website/pages/about.js

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable react/no-children-prop */
 import matter from 'gray-matter'
 import ReactMarkdown from "react-markdown";
+// @ts-ignore
```

Comment:
An alternative for this is to make the vertical-lines a javascript file and also let it receive props

Reply:
See the icons/github.js for more info

Reply:
Done ✅